clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -64,22 +81,42 @@ class AzureResourceManagerClient extends AzureBaseClient {
     deployment
   }
 
-  ResourceGroup createResourceGroup(AzureCredentials creds, String resourceGroupName, String region) {
+  /**
+   *
+   * @param creds
+   * @param resourceGroupName
+   * @param region
+     * @return
```

Comment:
Is the indentation of the `@return` annotation a preference thing? I haven't seen that before.